scout/build/case.py

```diff
@@ -205,12 +205,20 @@ def build_case(case_data, adapter):
     if case_data.get("rank_score_threshold"):
         case_obj["rank_score_threshold"] = float(case_data["rank_score_threshold"])
 
+    # Cohort information
+    if case_data.get("cohorts"):
+        case_obj["cohorts"] = case_data["cohorts"]
```

Comment:
I guess we should add new ones to the institute list as well?

Reply:
Yes I was wondering the same, I'll fix!

Reply:
Done, read below!